reader.py

```diff
@@ -0,0 +1,38 @@
+import http.client
+import json
+import datetime
+from util import SortUtil
+
+#Reads the data from the resources
+class Reader:
+    global start 
```

Comment:
Avoid using a global variable. https://betterprogramming.pub/alternatives-to-using-globals-in-python-a3b2a7d5411b

Reply:
Global variable has been removed